From 593ccc0cc8adc311fff05535d7dad57a6f4b4bc8 Mon Sep 17 00:00:00 2001
Message-ID: <593ccc0cc8adc311fff05535d7dad57a6f4b4bc8.1699403992.git.sharlatanus@gmail.com>
In-Reply-To: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
References: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 28 Oct 2023 03:51:47 +0100
Subject: [PATCH 18/67] gnu: go-github-com-jcmturner-dnsutils-v2: Adjust
 inputs.

* gnu/packages/golang-web.scm (go-github-com-jcmturner-dnsutils-v2):
Swap propagated-inputs to native-inputs.

Change-Id: I1d02b45559214d72828a78437766cf7c1e06f1b0
---
 gnu/packages/golang-web.scm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 9b2bcd7cbd..6a832291ab 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -302,7 +302,7 @@ (define-public go-github-com-jcmturner-dnsutils-v2
     (build-system go-build-system)
     (arguments
      '(#:import-path "github.com/jcmturner/dnsutils/v2"))
-    (propagated-inputs
+    (native-inputs
      (list go-github-com-stretchr-testify))
     (home-page "https://github.com/jcmturner/dnsutils")
     (synopsis "Go library with DNS utils")
-- 
2.41.0

